vega_sim/wallet/slim_wallet.py

```diff
@@ -17,6 +18,9 @@
 
 logger = getLogger(__name__)
 
+dotenv.load_dotenv()
```

Comment:
By a similar token to the below, let's move the dotenv load inside the `__init__`s of the services so we only modify someone's env when the sim is doing something which needs it.